roles/kube_nfs_volumes/tasks/nfs.yml

```diff
@@ -3,6 +3,13 @@
   action: "{{ ansible_pkg_mgr }} name=nfs-utils state=present"
   when: not openshift.common.is_containerized | bool
 
+- name: Work around https://bugzilla.redhat.com/show_bug.cgi?id=1394395
+  file: path=/var/lib/nfs/{{ item }}
+  with_items:
+    - etab
+    - rmtab
+  when: openshift.common.is_atomic | bool
+
```

Comment:
Thanks.  I am trying to make Atomic Host a fully general system and not just a kube node, for various reasons, so hosting NFS is definitely something I want to do.  The workaround is pretty small, and we'll fix it upstream ideally soon.